cmf-cli/resources/template_feed/grafana/Grafana.Package/README.md

```diff
@@ -0,0 +1,31 @@
+# Grafana custom Package Datasources
+
+If you want to know more about data sources in general [Official Documentation here!](https://grafana.com/docs/grafana/latest/datasources/)
+
+This information is for local development only!
+
+In order to create custom data sources or dashboards you should put them in the respective folders.
+
+If you are building an app and want to use the gRPC Data Manager data source you can use the following configuration:
```

Comment:
I'd change this sentence to the following:

If you are building an app and want to use the gRPC Data Manager data source you can `import` the following configuration `into your local grafana`: